Source/DomainObjects/AuthState.swift

```diff
@@ -90,37 +90,6 @@ public extension AuthStatus {
             return raw
         }
     }
-
-    public var description: String {
```

Comment:
I would mark this as deprecated before version 2.x.x

Reply:
Done.